### ISSUE TITLE
Adjust table font size per accessibility requirements

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/GEAR3#1015]
From the Customer Engagement team's [review of GEAR](https://docs.google.com/document/d/11GhUmVKvWu3Yy-D1chzo0iphUSnD2WeffN1oth9ymAo/edit?tab=t.0):

Make sure table rows have font size >= 12 pt

Column headings in tables should be 2 pt larger than the rows. Text is currently 12px, and it should be 14px. 

This is a requirement of 508 and WCAG (Web Content Accessibility Guidelines).

**Comment by shahasshrestha** [comment]
PR merged to fwdDev, https://github.com/GSA/GEAR3/pull/1019
